python/ray/train/v2/_internal/state/state_manager.py

```diff
@@ -120,7 +120,6 @@ def update_train_run_errored(
         run.end_time_ns = _current_time_ns()
         self._create_or_update_train_run(run)
 
-    # TODO: This may be handled in the StateManager.
```

Comment:
Note: May want to add a status detail here (for example to differentiate between being aborted due to Ctrl+C or something else)

Reply:
Can do in the second PR where it makes sense to have different definitions of ABORTED

Reply:
Done in other PR.